src/Umbraco.PublishedCache.HybridCache/Services/MediaCacheService.cs

```diff
@@ -295,5 +282,10 @@ private HybridCacheEntryOptions GetEntryOptions(Guid key)
         LocalCacheExpiration = _cacheSettings.Entry.Media.SeedCacheDuration,
     };
 
-    private string GetCacheKey(Guid key, bool preview) => preview ? $"{key}+draft" : $"{key}";
+    private static string GetCacheKey(Guid key, bool preview) => preview ? $"{key}+draft" : $"{key}";
+
+    // Generates the cache tags for a given CacheNode
+    // We use the tags to be able to clear all cache entries that are related to a given content item.
+    // Tags for now are content/media, draft/published and all its ancestors, so we can clear when ChangeType.TreeChange
```

Comment:
I don't quite follow this comment - the part _"draft/published and all its ancestors"_.  Seems we only have "content" and "media" as tags.  Could you clarify this please?

Reply:
Ah this in old comment, we decided not to have draft/published tags yet, I will change the comment 🙈